src/Chapter2.hs

```diff
@@ -332,11 +332,17 @@ yummy functions.
 
 Don't forget that you can load this module in GHCi to call functions
 from it!
+    
+3 5 0,1,2,3,4,5,6,7
 
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b l
+ | a < 0 || b < 0 = []
+ | a > b = []
+ | (length l) < a = []
+ | otherwise = take (b-a+1) (drop a l)
```

Comment:
Right! 👍🏼 

In Haskell, it is always better to separate operators and arguments with spaces 🙂 
```suggestion
 | otherwise = take (b - a + 1) (drop a l)
```